packages/release-tool/src/index.ts

```diff
@@ -398,13 +398,53 @@ function formatChangelog(previousReleasedVersion: string, prs: ExtendedGithubPrD
   ].join("\n");
 }
 
+async function getCommitsOfPr(pr: ExtendedGithubPrData) {
+  const commits: components["schemas"]["commit"][] = [];
+
+  for (let page = 1; ; page += 1) {
+    const pageResult = await octokit.request("GET /repos/{owner}/{repo}/pulls/{pull_number}/commits", {
+      owner: "lensapp",
+      repo: "lens",
+      pull_number: pr.number,
+      headers: {
+        'X-GitHub-Api-Version': '2022-11-28'
+      },
+      per_page: 100,
+      page,
+    });
+
+    commits.push(...pageResult.data);
```

Comment:
data is defined and empty if there are no commits?

Reply:
According to the typings from the GH package yes